src/logger.js

```diff
@@ -35,7 +35,7 @@ class CustomConsole extends Transport {
     // styles a console log statement accordingly to the log level
     // log level colors are taken from levelcolors enum
     // eslint-disable-next-line no-console
-    console.log(
+    console[info.level === "error" ? "error" : "log"](
```

Comment:
this broke some tests
if you were having trouble seeing logs -- it's probably because `npm run build` will strip console.log from the build
while `npm run build:watch` builds in debug and does not do this.

Reply:
I was running in build:watch and getting logs, but the logs had no stack trace. It would just log there was an error, but not where.

console.error emits a stack trace, but maybe something about the way this class is set up nests it so deep that it's not helpful.

Reply:
Want me to rever this?

Reply:
fixed